screens/SearchScreen.js

```diff
@@ -1,18 +1,16 @@
 import * as React from 'react';
 import { StyleSheet, Text, View } from 'react-native';
-import ShareIcon from '../components/share-app-component';
 
 const SearchScreen = () => {
   return (
     <View style={styles.container}>
       <Text> Search Screen </Text>
-      <ShareIcon/>
     </View>
   );
 };
 
+export default SearchScreen;
+
 const styles = StyleSheet.create({
-  container: {},
+  container:{}
 });
```

Comment:
This is not used so we can remove it as well.

Reply:
its on the search screen we don't have to remove that because we will have to have that there for the styling when we have all the components. So its basically the skeleton

Reply:
yup, that´s why I left it there.

Reply:
I just said it because as far as I remember, every pull request deals with only one single problem. So if the styles belongs to search screen, that should be done in that specific ticket not here. Correct me if I remember wrong please.

Reply:
Yes that is true but the style "container" was added right at the beginning with the setup of the project. Patrick didnt add it there, it's just showing he did a change there because he removed the share icon and that probably made it move on which line it was written. So basically it should have been there on all screens that were done on initial setup.